x-pack/plugin/relevance-search/src/test/java/org/elasticsearch/xpack/relevancesearch/query/RelevanceMatchQueryIntTests.java

```diff
@@ -43,7 +44,8 @@ public void setUp() throws Exception {
         relevanceMatchQueryBuilder = new RelevanceMatchQueryBuilder();
         relevanceMatchQueryBuilder.setQueryRewriter(getInstanceFromNode(RelevanceMatchQueryRewriter.class));
 
-        createIndex(DOCUMENTS_INDEX);
+        // We need to ensure the .ent-search index has been created, as the plugin might not have initialized it yet
```

Comment:
Why can't we initialize the plugin as a precondition?

Reply:
I had some interesting race conditions when relying on the plugin to create the index. It's probably because other thread groups are used, but I couldn't reliably wait for the plugin to create the index. This is the solution I have for now, though I would love to understand more about the init process in testing / in general to fix this properly.